vexflow/vexflow.d.ts

```diff
@@ -0,0 +1,1325 @@
+        scale(x : number, y : number) : IRenderContext;
+        resize(width : number, height : number) : IRenderContext;
+        fillRect(x : number, y : number, width : number, height : number) : IRenderContext;
+        clearRect(x : number, y : number, width : number, height : number) : IRenderContext;
+        beginPath() : IRenderContext;
+        moveTo(x : number, y : number) : IRenderContext;
+        lineTo(x : number, y : number) : IRenderContext;
+        bezierCurveToTo(x1 : number, y1 : number, x2 : number, y2 : number, x : number, y : number) : IRenderContext;
+        quadraticCurveToTo(x1 : number, y1 : number, x2 : number, y2 : number) : void;
+        arc(x : number, y : number, radius : number, startAngle : number, endAngle : number, antiClockwise : boolean) : IRenderContext;
+        glow() : IRenderContext;
+        fill() : IRenderContext;
+        stroke() : IRenderContext;
+        closePath() : IRenderContext;
+        fillText(text : string, x : number, y : number) : IRenderContext;
+        save() : IRenderContext;
+        restore() : IRenderContext;
+        
+        /**
+         * canvas returns TextMetrics, SVG returns SVGRect, Raphael returns {width : number, height : number}. Only width is used throughout VexFlow.
+         */
+        measureText(text : string) : {width : number};
+    }
+    
+    /**
+     * Helper interface for handling the Vex.Flow.Font object in Vex.Flow.Glyph. Not part of VexFlow!
+     */
+    interface IFont {
+        glyphs : {x_min : number, x_max : number, ha : number, o : string[]}[];
+        cssFontWeight : string;
+        ascender : number;
+        underlinePosition : number;
+        cssFontStyle : string;
+        boundingBox : {yMin : number, xMin : number, yMax : number, xMax : number};
+        resolution : number;
+        descender : number;
+        familyName : string;
+        lineHeight : number;
+        underlineThickness : number;
+        
+        /**
+         * This property is missing in vexflow_font.js, but present in gonville_original.js and gonville_all.js.
+         */
+        original_font_information? : {postscript_name : string, version_string : string, vendor_url : string, full_font_name : string, font_family_name : string, copyright : string, description : string, trademark : string, designer : string, designer_url : string, unique_font_identifier : string, license_url : string, license_description : string, manufacturer_name : string, font_sub_family_name : string};
+    }
+    
+    module Flow {
+        
+        var RESOLUTION : number;
+        
+        // from tables.js:
+        var STEM_WIDTH : number;
+        var STEM_HEIGHT : number;
+        var STAVE_LINE_THICKNESS : number;
+        var TIME4_4 : {num_beats : number, beat_value : number, resolution : number};
+        var unicode : {[name : string] : string}; //inconsistent API: this should be private and have a wrapper function like the other tables
```

Comment:
I don't use these on my project so I don't know if it is useful (or safe) to change these values.
But if these variables should be private, perhaps you can leave them as `const` to avoid accidental mutations by clients?